Geometry/TrackerCommonData/python/cmsExtendedGeometry2021XML_cfi.py

```diff
@@ -0,0 +1,385 @@
+import FWCore.ParameterSet.Config as cms
+
+# This config was generated automatically using generate2021Geometry.py
+# If you notice a mistake, please update the generating script, not just this config
```

Comment:
How did you generate this file? Didn't you have to change `dict2021Geometry.py` first? I'm not seeing that file in a PR.

Reply:
This must be a test copy - the geometry generation scripts use Geometry/CMSCommonData, not Geometry/TrackerCommonData

Reply:
OK. That's why I am confused. @bsunanda Please delete this comment if it is incorrect.

Reply:
Sure I shall do that. With this test I can know the changes are OK so that I can move to official area next